src/backend/distributed/operations/shard_cleaner.c

```diff
@@ -56,7 +116,7 @@ citus_cleanup_orphaned_shards(PG_FUNCTION_ARGS)
 	PreventInTransactionBlock(true, "citus_cleanup_orphaned_shards");
 
 	bool waitForLocks = true;
-	int droppedShardCount = DropOrphanedShards(waitForLocks);
+	int droppedShardCount = DropOrphanedShardsForMove(waitForLocks);
```

Comment:
we could also run cleanup from this function to use it in tests

Reply:
Yes I am using run_try_drop_marked_shards